installation/developer-installation/octobot-developer-installation.md

```diff
@@ -207,3 +206,121 @@ this script will install the OctoBot-Tentacles folder and run Octobot with all y
 * **Script to zip, install and run in one go**
 
 ![](<../../.gitbook/assets/image (12).png>)
+
+
+## Setting up Visual Studio Code IDE
+
+### Open project folder in VS Code
+
+Open the OctoBot Project root folder with VS Code
+
+### Create Virtual Environment
+
+{% hint style="info" %}
+depending your system instead of "python" you might need to use "python3.8" or "python38" . Especially if you have multiple versions of python installed it's important to use the right version
+{% endhint %}
+
+```bash
+python -m venv venv  
+```
+
+### Activate virtual environment
+
+```bash
+.\venv\Scripts\activate
+```
+
+### Create VSCode launch.json
+
+![](../../.gitbook/assets/image.png)
+
+{% hint style="info" %}
+The VS Code launch config is locate in project-folder/.vscode/launch.json
+{% endhint %}
+
+```json
+{
+    "version": "0.2.0",
+    "configurations": [
+
+        {
+            "name": "Start OctoBot",
+            "type": "python",
+            "request": "launch",
+            "env": {
+                "PYTHONPATH": "${workspaceFolder}/Async-Channel;${workspaceFolder}/OctoBot-Tentacles-Manager;${workspaceFolder}/OctoBot-Commons;${workspaceFolder}/OctoBot-Trading;${workspaceFolder}/OctoBot-Backtesting;${workspaceFolder}/OctoBot-evaluators;${workspaceFolder}/OctoBot-Services;${workspaceFolder}/trading-backend"
```

Comment:
I don't think we should add trading-backend in the developer env. Users should not modify edit.